vbfml/scripts/preprocess_image.py

```diff
@@ -0,0 +1,437 @@
+#!/usr/bin/env python3
+import glob
+import pickle
+import uproot
+import os
+import numpy as np
+import click
+import random
+import pandas as pd
+import matplotlib.pyplot as plt
+import awkward as ak
+
+from math import ceil
+
+from vbfml.training.plot import ImagePlotter
+from vbfml.util import get_process_tag_from_file
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.command()
+@click.option(
+    "-i",
+    "--input-files",
+    required=True,
+    help="Path to the ROOT file.",
+)
+def rotate(input_files: str):
+    """
+    Preprocess the image (in the root file) with a phi-rotation and eta-inversion to have the leading jet in the right center
+    """
+
+    # location and name of new root file -> in a new directory "_preprocessed"
+    output_dir = os.path.dirname(input_files) + "_preprocessed/"
```

Comment:
You can use f-strings: `output_dir = f'{os.path.dirname(input_files)}_preprocessed'`

Reply:
done